src/utils/organizationTagsUtils.ts

```
@@ -75,7 +75,8 @@ interface InterfaceBaseFetchMoreOptions<T> {
 }
 
 // 4. Query interfaces
-export interface InterfaceOrganizationTagsQuery extends InterfaceBaseQueryResult {
+export interface InterfaceOrganizationTagsQuery
+  extends InterfaceBaseQueryResult {
```

ESLint

[error] Delete `⏎·`

(prettier/prettier)